src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -53,7 +54,7 @@ public IntakeSubsystem() {
     m_pivotMotor.enableSoftLimit(SoftLimitDirection.kReverse, true);
     m_pivotMotor.setSoftLimit(SoftLimitDirection.kForward, (float) DEPLOY_SETPOINT);
     m_pivotMotor.setSoftLimit(SoftLimitDirection.kReverse,(float) STOW_SETPOINT);
-
+    m_pivotMotor.setSmartCurrentLimit(INTAKE_CURRENT_LIMITS);
```

Comment:
You never set the current limit for the intake motor.  Also, there should be a second constant for the intake motor.